### PR TITLE
Feature/generator

### PR DESCRIPTION
**Pull request by susch19**
Implemented a custom generator for property changed to have the ability to include elsa property changed later

### CODE REVIEW

File: AppBroker.Generators/AppBroker.Generators.csproj [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<LangVersion>9.0</LangVersion>
+
+		<!-- Special properties for analyzer packages. -->
+		<IncludeBuildOutput>false</IncludeBuildOutput>
+		<!--<IncludeSymbols>false</IncludeSymbols>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <UseNETCoreGenerator>true</UseNETCoreGenerator>
+    <IsRoslynComponent>true</IsRoslynComponent>-->
```
Comment (TheBlubb14) [inline AppBroker.Generators/AppBroker.Generators.csproj:12]:
```suggestion
```

File: AppBroker.Generators/AppBroker.Generators.csproj [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.10.0" PrivateAssets="all" GeneratePathProperty="true" />
+		<None Include="$(PkgMicrosoft_CodeAnalysis_CSharp)\lib\netstandard2.0\*.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+		<None Include="$(OutputPath)$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+	</ItemGroup>
+
+   <PropertyGroup>
+        <GetTargetPathDependsOn>$(GetTargetPathDependsOn);BlubbiChanged-GetDependencyTargetPaths</GetTargetPathDependsOn>
```
Comment (TheBlubb14) [inline AppBroker.Generators/AppBroker.Generators.csproj:26]: BlubbiChanged?

File: AppBroker.Generators/NotifyPropertyChangedGenerator.cs [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+        }}
+            ");
+
+
+
+            //foreach (IPropertySymbol propSymbol in thingy.Properties)
+            //{
+            //    ProcessProp(source, propSymbol);
+            //}
+
```
Comment (TheBlubb14) [inline AppBroker.Generators/NotifyPropertyChangedGenerator.cs:166]:
```suggestion
```

File: AppBroker.Generators/Properties/launchSettings.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (TheBlubb14) [inline AppBroker.Generators/Properties/launchSettings.json:1]: glaube sollte nicht eingecheckt werden

**Review by TheBlubb14 — CHANGES_REQUESTED** [review]

File: AppBroker.Generators/AppBroker.Generators.csproj (hunk 2 above)
Comment (susch19) [reply AppBroker.Generators/AppBroker.Generators.csproj:26]: Copied from you, doesn't have any effect, so keep it as a credit ;)

**Pull request merged**